src/Virgil.vfj

```diff
@@ -26309,83 +26318,10 @@
           }
         ]
       },
-      {
-        "name": "Omega",
```

Comment:
My fix for Omega and Delta is removed.

Reply:
Yes, we are still in the process of getting Ellinor up to speed with GitHub! Sorry about that.. :)

Your fixes were basically never fetched on her computer before she started fixing the issues! We accepted your PR sooner because you are @thorn0!